src/glb-director/tests/test_director_classify_ranges_v4.py

```diff
@@ -42,6 +43,10 @@ def get_initial_forwarding_config(cls):
 		}
 
 	def test_01_ip_range_match_v4(self):
+        # We don't yet support ip ranges in XDP
```

Comment:
Looks like this line has some tabs vs spaces going on.

Reply:
Very possible. My vim config has `set expandtab` while it is all tabs in the golang code. I'll fix those. Also, I think adding a check to see if the number of binds when expanded stays within the current static limit of 4096.
I'm going to add some of those.